### PR TITLE
Fix cellsu import

### PR DESCRIPTION
**Pull request by richardicooper**
Also fix crash on Windows reading blank lines from .ins files.

### CODE REVIEW

File: bits/shelx2cry/shelx_procedures.F90 [hunk 1]
```diff
@@ -446,10 +447,16 @@ subroutine shelx_zerr(shelxline)
...
+    
+    
     write (list31(1), '(a)') '\LIST 31'
-    write (list31(2), '("MATRIX V(11)=",F0.5, ", V(22)=",F0.5, ", V(33)=",F0.5)') esds(2:4)
-    write (list31(3), '("CONT V(44)=",F0.5, ", V(55)=",F0.5, ", V(66)=",F0.5)') esds(5:7)
-    write (list31(4), '(a)') 'END'
+    write (list31(2), '(a)') 'AMULT 0.000001'  ! Make this multiplier explicit, in case the default ever changes.
+    write (list31(3), '("MATRIX V(11)=",F0.5, ", V(22)=",F0.5, ", V(33)=",F0.5)') (esds(2:4)**2)/0.000001
+    write (list31(4), '("CONT V(44)=",F0.5, ", V(55)=",F0.5, ", V(66)=",F0.5)') ((esds(5:7)*DTR)**2)/0.000001
+    write (list31(5), '(a)') 'END'
```
Comment (pascal22p) [inline bits/shelx2cry/shelx_procedures.F90]: This is risky, you should use double precision for this.

double precision, parameter :: dtr = 0.01745329252d0
double precision, dimension(7) :: esds

(esds(2:4)**2)/0.000001d0
((esds(5:7)*DTR)**2)/0.000001d0

Comment (richardicooper) [reply bits/shelx2cry/shelx_procedures.F90]: Have fixed and at the same time spotted an error in one of the reference files:
eb2006.ins ZERR card start 2.0002.
This should be ZERR 2 .0002  - consistent with space group and expected magnitude of cell errors. Should we fix it or leave it in as a test case? (Shelx2cry reads the esds backwards to mitigate a missing Z value, but I don't think this is a valid SHELX syntax).

Comment (pascal22p) [reply bits/shelx2cry/shelx_procedures.F90]: There should be at least a warning that Z is missing. If you fixed the ins file, there should be an other one with Z missing.

How is shelx reading the file? Would it read 2.0002 as Z=2 and sx=.0002? I hope not that would be very difficult to parse reliably.

**Comment by richardicooper** [comment]
I don't think it is valid for shelx (at least it won't split the 2 from the
.0002. Will test.

**Pull request merged**